### ISSUE TITLE
Floats excluded from allowing tie downs

### ISSUE THREAD

**Issue by wlbragg** [c172p-team/c172p#1270]
@legoboyvdlp @gilbertohasnofb
The tiedowns-can-have property limits certain gear undercarriages from being able to use the tie downs. Anyone know why the bushkit 3 (floats) is not allowed to have tie-downs? I understand no chocks for greater than 2, but the tie-downs should work on all aircraft variants, yes?

Also, I have seen instances where a high wind can cause issues with the tie-downs or using the ctrl+u. I propose we use a tie-down-max-stress factor and remove the tie-downs it that stress factor is exceeded (in effect the tie-down breaks).

**Comment by legoboyvdlp** [comment]
Probably it assumes that it should be using the mooring instead of tie-downs?

**Comment by gilbertohasnofb** [comment]
Floats are only available on water, so what would the tie downs be attached
to? Amphibian has them since it can operate on land.

**Comment by legoboyvdlp** [comment]
Yeah, exactly - the floats should use mooring, not tie downs I think.

**Comment by wlbragg** [comment]
>Floats are only available on water, so what would the tie downs be attached to? Amphibian has them since it can operate on land.

Not tied down in the water, tie downs would be attached to the same thing as always, that doesn't change because there is a float undercarriage. I never really thought about the fact that the float may not be on a tarmac, only on the water. But, there is no adverse affect to make the option available if the owner of the aircraft wanted to work on the float undercarriage at the airport and have the aircraft ties down just in case? I think the option should still be there?

**Comment by legoboyvdlp** [comment]
Here's a picture of some tie downs - according to the page I found this on it uses small anchors.

http://www.pilotfriend.com/training/flight_training/fxd_wing/images4/26.jpg

Maybe if hydro system is active it could use anchors and swap to regular tie downs if it goes to asphalt? Notice this is distinct from mooring to an anchored buoy (the mooring system we already have).

**Comment by gilbertohasnofb** [comment]
I find that confusing. I would rather do the following:

* if hydro is active, we grey out tie downs and allow anchor
* if hydro is inactive, we grey out anchor and allow tie downs

 > But, there is no adverse affect to make the option available if the 
owner of the aircraft wanted to work on the float undercarriage at the 
airport and have the aircraft ties down just in case? I think the option 
should still be there?

I don't see much benefit at all, since you can't drag your plane to land 
in FlightGear as you could in RL. I am fine with removing the condition 
though, but I would feel strongly about greying it out or not according 
to the hydro activation status

**Comment by wlbragg** [comment]
>Here's a picture of some tie downs - according to the page I found this on it uses small anchors.

I have to admit, I really like the idea of being able to anchor the float/amphibious to a shore line mooring anchor when near the shore line. I may add it to the Cub and you guys can look at it and see if you like the way I implement it. I already figured out one way for sure to do this. Either manually with a button that says something like, "Set shoreline anchor" and when you hit that button it would pop up a prompt telling you to click on a spot on the shoreline to place the anchor. Or make it completely automatic and code places the anchor automatically, if it's possible to code. I don't know if there is enough topography data available to do this automatically.

>if hydro is active, we grey out tie downs and allow anchor

Basically, we already do this

>* if hydro is inactive, we grey out anchor and allow tie downs

Same here, I think we already do this.

>But, there is no adverse affect to make the option available if the owner of the aircraft wanted to work on the float undercarriage at the airport and have the aircraft ties down just in case? I think the option should still be there?

This was really my only point and it's not a big deal to me. i don't really see a use case  for loading FlightGear and tyeing down your floats at the airport so I guess it doesn't really matter.
I am changing the issue to Add shoreline mooring capability and allow tie-down failure under high stress.
